.env.example

```
@@ -15,5 +15,4 @@ LOG_LEVEL=debug
 LOGDNA_INGESTION_KEY=
 
 # payout variables
-UBIQUITY_BOT_EVM_PRIVATE_KEY=""
-RPC_PROVIDER_URL="https://example.com/"
\ No newline at end of file
+UBIQUITY_BOT_EVM_PRIVATE_KEY=""
```

```suggestion
EVM_PRIVATE_KEY=""
```

It probably makes sense to shorten because this is the only EVM private key of concern in this codebase, right?

---

The `UBIQUITY_BOT_EVM_PRIVATE_KEY` env variable is kind of deprecated. As soon as https://github.com/Alpha-Labs-Global team create an org wide config (with encrypted wallet private key) we are free to remove `UBIQUITY_BOT_EVM_PRIVATE_KEY` from our codebase and from netlify env variables.

---

Let's not wait on them.

---

It's been recently removed https://github.com/ubiquity/ubiquibot/pull/356